docs/source/welcome_to_sycamore/get_started.md

```diff
@@ -8,45 +8,47 @@ Sycamore is deployed using Docker, and you can launch it locally or on a virtual
 
 For more info:
 
-* Loading data into Sycamore
-* Querying your data
-* Using Jupyter notebook to customize data preparation code
+* [Loading data into Sycamore](/data_ingestion_and_preparation/load_data.md)
+* [Querying your data](/querying_data/demo_query_ui.md)
+* [Using Jupyter notebook to customize data preparation code](/data_ingestion_and_preparation/using_jupyter.md)
 
 *Note: By default, the Docker compose uses the stable version of the containers. You can choose a specific version to run, e.g. latest (last build pushed), latest_rc (last release candidate), or 0.YYYY.MM.DD (date-stamped release). To specify a version, set the VERSION environment variable, e.g. VERSION=latest_rc docker compose up --pull=always. See the .env file if you want to specify different versions for the separate containers.*
 
 ### Optional: Configure AWS Credentials for Amazon Textract usage
 
-Sycamore’s default data ingestion and preparation code can optionally use Amazon Textract for table extraction, which will give higher quality answers for questions on embedded tables in documents. To enable it, Sycamore needs AWS credentials for Amazon Textract and an Amazon S3 bucket for Textract input/output. The default code will use Textract and Amazon S3 in the US-East-1 region, and you will accrue AWS charges for Textract usage.
+Sycamore’s default data ingestion and preparation code can optionally use [Amazon Textract](https://aws.amazon.com/textract/) for table extraction, which will give higher quality answers for questions on embedded tables in documents. To enable it, Sycamore needs AWS credentials for Amazon Textract and an Amazon S3 bucket for Textract input/output. The default code will use Textract and Amazon S3 in the US-East-1 region, and you will accrue AWS charges for Textract usage.
 
-1. If you do not have an AWS account, sign up [here](https://portal.aws.amazon.com/billing/signup). You will need this during configuration.
+If you have starte Sycamore already, you'll need to restart it after following these instructions.
 
-2. Create an Amazon S3 bucket in your AWS account in the US-East-1 region for use with Textract (e.g. s3://username-textract-bucket). We recommend you set up bucket lifecycle rules that automatically delete files in this bucket, as the data stored here is only needed temporarily during a Sycamore data processing job.
+1. If you do not have an AWS account, sign up [here](https://portal.aws.amazon.com/billing/signup). You will need this during configuration. 
+
+2. Create an Amazon S3 bucket in your AWS account in the US-East-1 region for use with Textract (e.g. `s3://username-textract-bucket`). We recommend you set up bucket lifecycle rules that automatically delete files in this bucket, as the data stored here is only needed temporarily during a Sycamore data processing job. 
```

Comment:
us-east-1?  (AWS seems to use all lowercase in the UI)

Reply:
Ack